### PR TITLE
inspector 支持删除 events 的内容

### PR DESCRIPTION
**Pull request by youngfreeFJS**
inspector 支持删除 events 的内容
image

### CODE REVIEW

File: frontend/src/api/event.js [hunk 1]
```diff
@@ -45,3 +45,10 @@ export const getDefaultChannelNames = () => {
         url: '/api/channel/default'
     })
 }
+
+export const deleteEvents = () => {
+  return axios({
+    url: '/api//event',
```
Comment (yumiguan) [inline frontend/src/api/event.js:51]: double //

Comment (youngfreeFJS) [reply frontend/src/api/event.js]: fixed

**Review by zhaoye — APPROVED** [review]

File: frontend/src/views/inspector/ButtonBar.vue [hunk 2]
```diff
@@ -113,11 +113,18 @@
...
     <Modal
       v-model="showClearModal"
-      title="Alert"
+      title="Clear"
       @on-ok="clearAllFlow"
       @on-cancel="showClearModal=false"
     >
-      <p>Clear flow list?</p>
+      <CheckboxGroup v-model="clearTypes" size=default>
+        <Tooltip max-width="200" content="this operation will delete all your Real-time tab" placement="top-start">
```
Comment (yumiguan) [inline frontend/src/views/inspector/ButtonBar.vue:121]: This operation will delete all your Real-time flow.

File: frontend/src/views/inspector/ButtonBar.vue [hunk 3]
```diff
@@ -113,11 +113,18 @@
...
+          <Checkbox label="Real-time" border />
+        </Tooltip>
+        <Tooltip max-width="200" content="this operation will delete all your local database." placement="top-start">
```
Comment (yumiguan) [inline frontend/src/views/inspector/ButtonBar.vue:124]: This operation will drop your local database.

**Review by yumiguan — APPROVED** [review]

**Pull request merged**